homeassistant/components/icloud/config_flow.py

```diff
@@ -95,7 +102,13 @@ def __init__(self):
 
         try:
             self.api = await self.hass.async_add_executor_job(
-                PyiCloudService, self._username, self._password, icloud_dir.path
+                PyiCloudService,
+                self._username,
+                self._password,
+                icloud_dir.path,
+                True,
+                None,
```

Comment:
`True` and `None` are library default value to params.

I put them since I don't know how to use positional param with `self.hass.async_add_executor_job`

But, if needed I can remove all params after `icloud_dir.path` since it's not gonna change the checks after that.